task1/getUniqueNumber.js

```diff
@@ -0,0 +1,7 @@
+function getUnique(array) {
+  for (let i = 0; i < array.length; i++) {
+    if (array.indexOf(array[i]) == array.lastIndexOf(array[i])) return array[i];
```

Comment:
Can you please write if statement with the correct syntax:
```
if (array.indexOf(array[i]) == array.lastIndexOf(array[i])) {
  return ...
}
```
It will be more clear and redable for other developers